### PR TITLE
feat(value): create and impl ToJson trait

### PR DESCRIPTION
**Pull request by s-panferov**
Refs #62

What do you think? Tests fail on `nightly`, but this seems unrelated.

### CODE REVIEW

File: json/src/value.rs [hunk 1]
```diff
@@ -1001,3 +1001,15 @@ pub fn from_value<T>(value: Value) -> Result<T, Error>
...
+pub trait ToJson {
+    /// Converts the value of `self` to an instance of JSON
+    fn to_json(&self) -> Value;
+}
+
+impl<T: ?Sized> ToJson for T where T: ser::Serialize {
+    fn to_json(&self) -> Value {
+        to_value(&self)
+    }
+}
```
Comment (erickt) [inline json/src/value.rs]: It looks like there isn't a trailing newline at the end of the file. Could you add one?

**Comment by erickt** [comment]
This seems reasonable to me. r=erickt with my comment addressed.

File: json/src/value.rs (hunk 1 above)
Comment (s-panferov) [inline json/src/value.rs]: Sure

**Comment by oli-obk** [comment]
@erickt: @dtolnay and I don't have push access to the json repo

**Comment by oli-obk** [comment]
testing whether I have homu access. The travis failure is due to clippy

@homu r+

**Pull request merged**